llama_index/retrievers/image_retriever.py

```diff
@@ -0,0 +1,75 @@
+"""Base vector store index query."""
+
+from abc import abstractmethod
+from typing import Any, Dict, List, Optional
+
+from llama_index.constants import DEFAULT_SIMILARITY_TOP_K
+from llama_index.indices.base_retriever import BaseRetriever
+from llama_index.indices.query.schema import QueryBundle
+from llama_index.schema import NodeWithScore
+from llama_index.vector_stores.types import (
+    MetadataFilters,
+)
+
+
+class BaseImageRetriever(BaseRetriever):
+    """Base Image retriever Abastraction.
+
+
+    Args:
+        BaseImageRetriever
+        similarity_top_k (int): number of top k results to return.
+        filters (Optional[MetadataFilters]): metadata filters, defaults to None
+    """
+
+    def __init__(
+        self,
+        image_similarity_top_k: int = DEFAULT_SIMILARITY_TOP_K,
```

Comment:
lets kep the top-k kwarg the same -- `similarity_top_k`

Reply:
+1